src/main/kotlin/com/autonomousapps/extension/ProjectIssueHandler.kt

```diff
@@ -54,21 +55,26 @@ import javax.inject.Inject
  * ```
  */
 open class ProjectIssueHandler @Inject constructor(
-  private val name: String,
+  private val projectPath: String,
   objects: ObjectFactory
 ) : Named {
 
-  override fun getName(): String = name
+  override fun getName(): String = projectPath
 
-  internal val anyIssue = objects.newInstance(Issue::class.java)
-  internal val unusedDependenciesIssue = objects.newInstance(Issue::class.java)
-  internal val usedTransitiveDependenciesIssue = objects.newInstance(Issue::class.java)
-  internal val incorrectConfigurationIssue = objects.newInstance(Issue::class.java)
-  internal val unusedAnnotationProcessorsIssue = objects.newInstance(Issue::class.java)
-  internal val compileOnlyIssue = objects.newInstance(Issue::class.java)
-  internal val runtimeOnlyIssue = objects.newInstance(Issue::class.java)
-  internal val redundantPluginsIssue = objects.newInstance(Issue::class.java)
-  internal val moduleStructureIssue = objects.newInstance(Issue::class.java)
+  internal val sourceSets = objects.domainObjectContainer(
```

Comment:
I'm deliberately not documenting this because I don't want to encourage the general population to use the new API.